src/app/session.cpp

```diff
@@ -310,9 +311,16 @@ void Session::Forge::iconize(QString name, QAbstractButton *button, uint8_t size
 {
     auto scaled = size * this->arbiter_.layout().scale;
     button->setIconSize(QSize(scaled, scaled));
+    button->setIcon(QIcon(QString(":/icons/%1.svg").arg(name)));
```

Comment:
this could be refactored to call `set_icon` (no need for the duplicate code)

Reply:
yep. This is a relic.

Mildly interesting: the second case used to call code that dynamically changed the icon colors to work with both light and dark themes (it worked!) but then I realized that I could just tweak the icon to look good without modifications so I ripped it out in the interest of simplicity.